src/main/java/com/teamacronymcoders/base/modules/journeymap/JourneyMapPlugin.java

```diff
@@ -0,0 +1,29 @@
+package com.teamacronymcoders.base.modules.journeymap;
+
+import com.teamacronymcoders.base.Base;
+import journeymap.client.api.IClientAPI;
+import journeymap.client.api.IClientPlugin;
+import journeymap.client.api.event.ClientEvent;
+
+import static com.teamacronymcoders.base.Reference.MODID;
+import static net.minecraftforge.common.MinecraftForge.EVENT_BUS;
+
+@journeymap.client.api.ClientPlugin
```

Comment:
Any specific reason why this has the fully qualified name?

Reply:
Nope, I was following the JMApi example by rote instead of using my brain.